Makefile

```diff
@@ -87,7 +87,13 @@ tox-env-clean:
 	rm -rf .tox
 
 lint: check-venv
-	@find esrally benchmarks scripts tests it -name "*.py" -exec $(VEPYLINT) -j0 -rn --load-plugins pylint_quotes --rcfile=$(CURDIR)/.pylintrc \{\} +
+	@find esrally benchmarks scripts tests it -name "*.py" -exec $(VEPYLINT) -j0 -rn --rcfile=$(CURDIR)/.pylintrc \{\} +
+	@black --check esrally benchmarks scripts tests it
```

Comment:
```suggestion
	@. $(VENV_ACTIVATE_FILE); black --check esrally benchmarks scripts tests it
```

Reply:
Should I use VEBLACK for consistency with VEPYLINT above?

Reply:
Or use `VENV_ACTIVATE_FILE` for pylint too?

Reply:
We only use `VEPYLINT` above because of `find` so let's stick to `VENV_ACTIVATE_FILE` here.